### PR TITLE
Update CovarianceMatrices to v0.0.3

### PR DESCRIPTION
**Pull request by gragusa**
CovarianceMatrices: Heteroskedasticity and Autocorrelation Consistent Covariance Matrix Estimation for Julia.

### CODE REVIEW

File: CovarianceMatrices/versions/0.0.3/requires [hunk 1]
```diff
@@ -0,0 +1,5 @@
+julia 0.3+
+StatsBase
```
Comment (tkelman) [inline CovarianceMatrices/versions/0.0.3/requires]: Dont think the + is valid here

Comment (gragusa) [inline CovarianceMatrices/versions/0.0.3/requires]: Straggled one hour on choosing the right REQUIRE and still picked the wrong
one :)

Should it be 0.3? How do I fix this? Close PR, delete the remote tag, fix,
retag, and do a new PR?


Giuseppe Ragusa

Comment (tkelman) [inline CovarianceMatrices/versions/0.0.3/requires]: You can remove the + here and remove from package require separately. No need to re tag

**Comment by milktrader** [comment]
Have you considered contributing to JuliaStats/TimeModels?

**Comment by gragusa** [comment]
@milktrader Do you mean merging `CovarianceMatrices` into `TimeModels` or contributing in general? If you mean the latter, I could have something to contribute. If it is the former I am not sure covariance matrices are to be used in time series context. As such, it should live on a separate package.

**Pull request merged**